TestAssets/TestProjects/PortableApp/project.json

```diff
@@ -9,7 +9,7 @@
       "dependencies": {
         "Microsoft.NETCore.App": {
           "type": "platform",
-          "version": "1.0.0"
+          "version": "1.0.0-servicing-004500"
```

Comment:
@brthor, from a host testing standpoint, any app should be fine. We will apply roll-forward for portable apps between pre-release to pre-release (but not to production), so this PJ doesn't need to be updated. The reason it would fail if you had `1.0.0` was host roll-forward doesn't go from pre-release to production by design for portable apps. If you had `1.0.0-servicing-*` NuGet always resolves to `1.0.0` hitting the same roll-forward issue.

If you are worried about testing of the just-built `Microsoft.NETCore.App` as well -- it is broken for standalone apps as NuGet resolves it from the feed with `1.0.0` and not `1.0.0-servicing-*`.